snuba/datasets/entity.py

```diff
@@ -72,6 +74,9 @@ def get_all_join_relationships(self) -> Mapping[str, JoinRelationship]:
         """
         return self.__join_relationships
 
+    def get_required_conditions(self) -> Mapping[str, Sequence[str]]:
+        return self.__required_conditions
+
```

Comment:
Shouldn't this allow multiple condition requested for the same field? Like `timestamp` requiring a GTE and a LT condition to ensure we have an interval in the query.

Reply:
Why are both necessary? Only one is needed to create a time window < 90 days.

Reply:
They are desirable so that clients have to be deliberate in picking a closed time range. In one way is to filter off spurious events that have a timestamp in the future (they exist, not kidding). And on the other hand is for when they want a short time range in the past (like 12 hours of one day 2 months ago). Having to provide the close interval pushes the client to think what is the shortest time range they need.